### PR TITLE
ft-api-user-get-all-received-emails, add route to get all messages [finishes #164727356]

### PR DESCRIPTION
**Pull request by Ivankivu**
This PR adds the api route for get-all-messages

### CODE REVIEW

File: app/api/models/messages.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+data = []
+message_id = Validator.auto_id(dict, messages)
+parentMessageId = Validator.auto_id(dict, messages)
+createdOn = date
+
+
+class Message:
+
+    def __init__(self, **kwargs):
+        self.message_id = kwargs.get("message_id")
```
Comment (mendozabree) [inline app/api/models/messages.py:17]: Could you please be consistent in your variable naming.
here is a [resource](https://medium.com/@dasagrivamanu/python-naming-conventions-the-10-points-you-should-know-149a9aa9f8c7)

Comment (Ivankivu) [reply app/api/models/messages.py:17]: Okay. Will work towards changing that.

File: app/api/views/viewusers.py [hunk 2]
```diff
@@ -18,11 +20,53 @@ def signup():
...
+            if user["email"] != email and user["password"] != password:
+                return jsonify({'message': "login failed!!"}), 400
+            else:
+                return jsonify({'message': "logged in"}), 200
+
+
+class ViewMessage(MethodView):
+
+    @app.route('/api/v1/messages', methods=['POST'])
+    def new_message():
```
Comment (mendozabree) [inline app/api/views/viewusers.py]: This function is repeated.

Comment (Ivankivu) [reply app/api/views/viewusers.py]: yes was just pushed changes.

**Pull request closed without merging**